### ISSUE TITLE
Jest encountered an unexpected token

### ISSUE THREAD

**Issue by repl-albert-chu** [lytc/react-sortly#6]
Im encountering this error when trying to test my project if I have sortly enabled

```
  ● Test suite failed to run

    Jest encountered an unexpected token

    This usually means that you are trying to import a file which Jest cannot parse, e.g. it's not plain JavaScript.

    By default, if Jest sees a Babel config, it will use that to transform your files, ignoring "node_modules".

    Here's what you can do:
     • To have some of your "node_modules" files transformed, you can specify a custom "transformIgnorePatterns" in your config.
     • If you need a custom transformation specify a "transform" option in your config.
     • If you simply want to mock your non-JS modules (e.g. binary assets) you can stub them out with the "moduleNameMapper" config option.

    You'll find more details and examples of these config options in the docs:
    https://jestjs.io/docs/en/configuration.html

    Details:

    C:\Workspaces\myProject\frontend\node_modules\react-sortly\lib\index.js:1
    ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){import Sortly from './Sortly';
                                                                                             ^^^^^^
```

**Comment by LyTranCong** [comment]
@repl-albert-chu That's because of wrong build. Fixed in `1.5.2`!

**Comment by repl-albert-chu** [comment]
Updated to the latest version (1.5.3) and the issue went away. Thanks for the quick fix!
